app/subscriber/src/components/layout/constants/MenuItemNames.ts

```diff
@@ -0,0 +1,13 @@
+// enum that controls the items that will appear in the navigation menu, please note that if you are adding a new item
+// you will need to add a custom icon for it in SelectableMenuItems.tsx otherwise it will use the default icon
+export enum MenuItemNames {
```

Comment:
Change this to an array of menu items.  Each menu item will be an object with a label, path, and icon.  This way the related information for each menu item is all in one place.

Reply:
Lol that makes a whole lot more sense, reading over what I did now made me chuckle